pkg/edgeview/README.md

```diff
@@ -0,0 +1,38 @@
+# Edgeview container in EVE
+
+## ./src directory
+
+It contains the main package of golang source for 'eve-edgeview' container on EVE device. The 'edge-view-init.sh' script is started with the container and in a loop for start/stop the edgeview program base on the configurations. The same script is run on the client side, the environment variable `EDGEVIEW_CLIENT` needs to be set and passed in.
+
+## ./dispatcher directory
+
+It contains an example of golang source for 'edgeview' websocket dispatcher running in the cloud or in some VPN servers
+
+## Dockerfiles
+
+The 'Dockerfile' in the directory is used for EVE to build the 'edgeview' container running on EVE device
+The same docker container `lf-edge/eve-edgeview` can be run on user laptops.
+
+## Query script
+
+The 'query-edgeview.sh' is an example of running the `lf-edge/eve-edgeview` docker container on the client computer
+
+## Makefile
+
+the 'Makefile' supports in pkg/edgeview directory:
+
+(1) 'make eve-edge-view' to build the lf-edge/eve-edge-view:latest docker container;
+
+(2) 'make wss-server' to build a golang program for 'edgeview' websocket dispatcher. It needs to be run this compile on a Linux server if the websocket dispatcher will run in the same architecture;
+
+## Help
+
+On edge-vew client container, or 'lf-edge/eve-edgeview', use '-h' or '-help' to see all the options, and one can do help on specific command option to get detail on the command.
+
+ options:
+  log/search-pattern [ -time start_time-end_time -json -type app|dev -extra num ]
+
+  pub/ [baseosmgr domainmgr downloader global loguploader newlogd nim nodeagent tpmmgr vaultmgr volumemgr watcher zedagent zedclient zedmanager zedrouter]
+
+  [acl app arp connectivity flow if mdns nslookup ping route socket speed tcp tcpdump trace url wireless]
+  [app configitem cp datastore download hw lastreboot model newlog pci ps cipher usb shell volume]
```

Comment:
Please remove shell and add the new commands.

Reply:
ok.